listen.py

```diff
@@ -38,7 +38,7 @@ def from_json(cls, j):
             user_name=j.get('user_name', ""),
             timestamp=datetime.utcfromtimestamp(float(j['listened_at'])),
             artist_msid=j['track_metadata']['additional_info'].get('artist_msid'),
-            album_msid=j['track_metadata']['additional_info'].get('album_msid'),
```

Comment:
I suspect it's because listenstore was originally written with `album`. The rest of listenbrainz used `release`, but we've probably been slowly building out features using listenstore as a base.
I'd confirm with @mayhem, but my opinion is that we should use the MusicBrainz standard here, which is release, not album.